docs/getting_support.rst

```diff
@@ -3,47 +3,461 @@
+
+In order to use this encrypted email workflow, you need to:
+
+1. Provide **your public key** to the support server
+2. Import the **server's public key** into your local GPG keyring.
+
+The following documentation explains how to perform these steps to get
+the encrypted email workflow working. This documentation assumes you are
+familiar with PGP/GPG. We encourage you to use your preferred GPG key
+management tool and email client.
+
+If you do not know how to do things like import and export GPG keys,
+the encrypted email workflow is not for you. We recommend you stick to
+the :ref:`web-based workflow <using_redmine>`.
+
+Providing your public key
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Start by navigating to
+`support.freedom.press/pgp <https://support.freedom.press/pgp>`_ or
+click on the "PGP" menu option in the upper right corner of the support window.
+
+|PGPMenu|
+
+In the PGP management window you will see two columns: one for your
+public key on the left, and one with the server's public key on the
+right.
+
+|PGPUpload|
+
+To provide your public key, start by exporting an ASCII-armored copy of
+your public key. Paste your armored public key into the empty text box
+on the left and hit **Save**.
+
+|PGPSave|
+
+Importing the server's public key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now, import the public key for our Redmine server into your local GPG
+keyring. It is available on the right hand side of the page in the
+column with the heading "Redmine Server (support@freedom.press)", in the
+text box labeled "Public PGP key".
+
+Select the entire public key and copy it to your clipboard. Import the
+copied public key into your local GPG keyring.
+
+You can `download the key <https://freedom.press/documents/37/redmine_public_key.asc>`_ 
+or fetch it from a `keyserver <https://keys.openpgp.org/vks/v1/by-fingerprint/D0E0B2F2B71BA4E48278037D9EA33029E9FBBA2E>`_ (fingerprint:
+``D0E0B2F2B71BA4E48278037D9EA33029E9FBBA2E``).
+
+
+.. todo:: Add section on testing encrypted email after setting it up
+
+.. |PGPMenu| image:: images/support/pgp_menu.png
+.. |PGPUpload| image:: images/support/pgp_upload.png
+.. |PGPSave| image:: images/support/pgp_save.png
+
+
+.. _community_support:
+
 Community Based Support
------------------------
+^^^^^^^^^^^^^^^^^^^^^^^
+
 The `SecureDrop forum <https://forum.securedrop.org/>`_ is a good place to
 discuss SecureDrop and to get help from the international community of
 SecureDrop users and developers.
 
 You can also connect directly with the SecureDrop development team and the larger
-SecureDrop community using the `SecureDrop Gitter channel <https://gitter.im/freedomofpress/securedrop>`_ .
+SecureDrop community using the
+`SecureDrop Gitter channel <https://gitter.im/freedomofpress/securedrop>`_ .
 
 .. warning::
 
    Remember that both the SecureDrop forum and the Gitter channel are
    public. **Do not post any sensitive information through public channels.**
+
+Troubleshooting
+^^^^^^^^^^^^^^^
```

Comment:
I think we'll need to tweak organization and section title here. What are we troubleshooting - Redmine access itself, or SecureDrop? If it's the latter, some of the sub-sections don't really fit. Given that this is a second-level heading, I could see folks getting confused -- "I'm trying to troubleshoot 2FA on SecureDrop, but this seems to be something else entirely"

The old troubleshooting section was always messy and will ultimately benefit from some cleanup. For now, I would suggest:
- Put critical onboarding and usage information in the first chapter (i.e. creating issues, setting up 2FA)
- Put support portal tips and troubleshooting information in the second chapter, under a title like "Support Portal Tips and Troubleshooting" or similar, that leaves now doubt about its purpose

Reply:
Thanks! I ended up restructuring this quite a lot, and in the end I realized the information in the current "Troubleshooting" section actually belonged within the main headings rather than a catch-all "Troubleshooting" section. As a result, as of the most recent commit, this section is now omitted entirely. Please let me know if you like this new structure better, and if you think it's necessary to have tips/troubleshooting section here at all.